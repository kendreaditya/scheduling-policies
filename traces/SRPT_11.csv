1,0,20
2,1,19
3,2,18
4,3,17
5,4,16
6,5,15
7,6,14
8,7,13
9,8,12
10,9,11
11,10,10
12,11,9
13,12,8
14,13,7
15,14,6
16,15,5
17,16,4
18,17,3
19,18,2
20,19,1
