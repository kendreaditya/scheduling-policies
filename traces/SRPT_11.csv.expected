1,20
2,210
3,191
4,173
5,156
6,140
7,125
8,111
9,98
10,86
11,75
12,65
13,56
14,48
15,41
16,35
17,30
18,26
19,23
20,21
